packages/opensrp-plans/src/ducks/index.ts

```diff
@@ -129,7 +129,7 @@ const initialState: ImmutablePlanDefinitionState = SeamlessImmutable({
  * @param {object} action - plans actions
  * @returns {object} - the updated states
  */
-export default function reducer(
+export function reducer(
```

Comment:
why this change?

Reply:
we are exporting several values. During the import, one will be able to use the same name to refer to the corresponding value. This seemed more explicit than having a default in a file with many exports

Reply:
I understand that.  However the Ducks: Redux Reducer Bundles "spec" states that each reducer module [must have a reducer function that is default exported](https://github.com/erikras/ducks-modular-redux#rules).  Does this change made here mean that we have decided that all our ducks will no longer default export the reducer function?  If so, why?

Reply:
In that case I think adhering to ducks spec would be most ideal